final_task/rss_reader/ConvertToHtmlAndPdf.py

```diff
@@ -0,0 +1,100 @@
+import os
+import sys
+from fpdf import FPDF
+from News import News
+from dominate.tags import html, head, meta, body, div, img, p, b, br, h1, a
+
+from RssException import RssException
+
+
+def convert_Dict_to_News(arr_news_dict):
+    all_news = []
+    for item_news in arr_news_dict:
+        tmp_img_link = item_news["links"]
+        tmp_link = item_news["link"]
+        tmp_news = item_news["news"]
+        tmp_title = item_news["title"]
+        tmp_date = item_news["date"]
+        tmp_date_str_date = item_news["strDate"]
+        item_of_list_news = News(tmp_news, tmp_link, tmp_title, tmp_date, tmp_img_link, tmp_date_str_date)
+        all_news.append(item_of_list_news)
+    return all_news
+
+
+def create_html_news(path, News=[]):
```

Comment:
During our course, we've explained why is it bad to use mutable types as default arguments in functions. Please, fix it